### PR TITLE
Configurable remote config poll interval

### PR DESCRIPTION
**Pull request by CarlesDD**
### What does this PR do?
Provides a way to configure the Remote Configuration Poll interval via environment variable.

### CODE REVIEW

File: index.d.ts [hunk 1]
```diff
@@ -521,6 +521,17 @@ export declare interface TracerOptions {
...
+
+  /**
+   * Configuration of ASM Remote Configuration
+   */
+  remoteConfig?: {
+    /**
+     * Specifies the remote configuration polling interval in seconds
+     * @default 5
+     */
+    pollInterval?: number,
```
Comment (simon-id) [inline index.d.ts:533]: You also need to add an entry to `docs/test.js` when adding new programmatic conf

**Review by simon-id — CHANGES_REQUESTED** [review]

File: index.d.ts (hunk 1 above)
Comment (simon-id) [reply index.d.ts:533]: basically just add that line 87
```js
remoteConfig: {
  pollInterval: 5
}
```

File: packages/dd-trace/test/appsec/remote_config/manager.spec.js [hunk 2]
```diff
@@ -47,7 +47,10 @@ describe('RemoteConfigManager', () => {
       },
       service: 'serviceName',
       env: 'serviceEnv',
-      version: 'appVersion'
+      version: 'appVersion',
+      remoteConfig: {
+        pollInterval: 5
```
[the remark this answers is not in the document]
Comment (CarlesDD) [reply packages/dd-trace/test/appsec/remote_config/manager.spec.js:52]: There is an [assertion](https://github.com/DataDog/dd-trace-js/blob/09d49ff1ec31ed239f4d3793c78d85ece8bbe3a8/packages/dd-trace/test/appsec/remote_config/manager.spec.js#L65) to check that the pollInterval config is passed to Scheduler. Are you referring to something else?

File: index.d.ts (hunk 1 above)
Comment (CarlesDD) [reply index.d.ts:533]: Done!

File: packages/dd-trace/test/appsec/remote_config/manager.spec.js (hunk 2 above)
[the remark this answers is not in the document]
Comment (simon-id) [reply packages/dd-trace/test/appsec/remote_config/manager.spec.js:52]: Oh ok, good enough then!

**Review by simon-id — APPROVED** [review]

**Pull request merged**